networkx/algorithms/link_analysis/tests/test_pagerank.py

```diff
@@ -83,11 +83,8 @@ def test_numpy_pagerank(self):
         for n in G:
             assert p[n] == pytest.approx(G.pagerank[n], abs=1e-4)
 
-    # This additionally tests the @nx._dispatchable mechanism, treating
-    # nx.google_matrix as if it were a re-implementation from another package
-    @pytest.mark.parametrize("wrapper", [lambda x: x, dispatch_interface.convert])
-    def test_google_matrix(self, wrapper):
-        G = wrapper(self.G)
+    def test_google_matrix(self):
+        G = self.G
```

Comment:
Yeah, that's exactly right. We include this one still for sanity. It's here, because it's where @jim22k added it ~2.5 years ago :joy:. Yeah, it would be nice to move it to `test_backends.py`... hopefully part of efforts to improve dispatching testing.